### PR TITLE
Fix/actions row break

### PR DESCRIPTION
**Pull request by mdpx**
## Related Issue

#94 

## Description

simply applied flex style to the ref wrapper, and removed the old flex box;
also eliminated some redundant code in m-table-body

## Impacted Areas in Application

MTableActions
MTableEditRow
m-table-body-row

m-table-body

## Additional Notes

also enabled two demo components

### CODE REVIEW

**Comment by peacechen** [comment]
@oze4 Would you mind merging and publishing this fix? Would like to restore the horizontal toolbar rendering.
🙇

**Review by matthewoestreich — APPROVED** [review]

**Comment by matthewoestreich** [comment]
Merged. If you don't see a new version by tomorrow afternoon please remind me. I'm not able to publish right now. 

I apologize, I must have missed this request.

**Comment by peacechen** [comment]
Thanks @oze4 

In order to avoid the single-person bottleneck of the original material-table project, I hope that you'll add collaborators who can merge, as well as npm collaborators who can help publish.

**Comment by peacechen** [comment]
ping @oze4

**Comment by matthewoestreich** [comment]
Hey I apologize. I wound up rescuing a dog spur of the moment and just got him out of the vet. I'll be by my computer in about an hour and will knock this out then. 

I also think we need more ppl with access to push/publish. I hate feeling like I'm letting people down and being a bottleneck. I would want @Domino987 input first. With that being said we don't want to give access to the wrong people so I'm not sure how I feel about it. 

If anyone is interested please let us know!

**Comment by matthewoestreich** [comment]
>In order to avoid the single-person bottleneck of the original material-table project, I hope that you'll add collaborators who can merge, as well as npm collaborators who can help publish.

@peacechen I must have overlooked that. Technically there are two of us but your point is still valid. You interested?

**Comment by peacechen** [comment]
I'd be happy to help out. My github username is `peacechen`, and my npm username is simply `peace`

**Comment by peacechen** [comment]
ping @oze4

**Comment by Domino987** [comment]
Hi sorry i am quite busy right now so it would be great to have more people on board. I think 4 would be a good number of people and I think we should have some kind of metric to choose. Maybe 4-5 commits to the repo to show some.kind of dedication? But I am open to suggestions. And as previous, requiring an approve from one other of the maintainers before merge?

**Comment by matthewoestreich** [comment]
@Domino987 I like that idea.

**Comment by matthewoestreich** [comment]
@peacechen v2.3.27 has been published.

**Comment by peacechen** [comment]
Thanks @oze4 🎉 

Since merges require an approver, it would be better to have more collaborators because that means 2 people are required to merge. That doubles the chances of people's schedules blocking merges. 4 collaborators essentially becomes the equivalent of 2 people.

**Comment by matthewoestreich** [comment]
I'm sure that's something we can change. Like you say, otherwise it kind of defeats the purpose.

**Pull request merged**
